cmd/monaco/integrationtest/assert.go

```diff
@@ -158,7 +158,7 @@ func assertConfig(t *testing.T, client dtclient.ConfigClient, theApi api.API, en
 }
 
 func assertSetting(t *testing.T, c dtclient.SettingsClient, typ config.SettingsType, environment manifest.EnvironmentDefinition, shouldBeAvailable bool, config config.Config) {
-	expectedExtId := idutils.GenerateExternalID(typ.SchemaId, config.Coordinate.ConfigId)
+	expectedExtId, _ := idutils.GenerateExternalID(config.Coordinate)
```

Comment:
I think in the error case, the assert will not behave wrongly won't it?

If there's an error the id is empty string,  so the filter will include all Settings objects that don't  have an externalId attached. 

Would suggest to error out of the assert if generating the ID failed